src/typing_extensions.py

```diff
@@ -1102,6 +1084,9 @@ class Film(TypedDict):
             is_typeddict(Film)  # => True
             is_typeddict(Union[list, str])  # => False
         """
+        # On 3.8, this would otherwise return True
+        if hasattr(typing, "TypedDict") and tp is typing.TypedDict:
+            return False
         return isinstance(tp, tuple(_TYPEDDICT_TYPES))
```

Comment:
It's already a tuple

```suggestion
        return isinstance(tp, _TYPEDDICT_TYPES)
```

Reply:
Thanks, pushed